packages/modular-scripts/src/__tests__/index.test.ts

```diff
@@ -251,6 +258,18 @@ describe('modular-scripts', () => {
       }
     }
 
+    if (port) {
+      // kill all processes listening to the dev server port
+      exec(
+        `lsof -n -i4TCP:${port} | grep LISTEN | awk '{ print $2 }' | xargs kill`,
```

Comment:
I would guess if we ever add Windows test support this will crop up then...

Reply:
Yeah - this will need to be updated for Windows test support. We want to redo these tests eventually as well to cleanly start and finish every time.